gensrc/proto/lake_service.proto

```diff
@@ -121,6 +121,22 @@ message UnlockTabletMetadataResponse {
     optional int32 pad = 1;
 }
 
+message UploadSnapshotsRequest {
+    map<int64, Snapshot> snapshots = 1;
+    optional string broker = 2;
```

Comment:
no necessary for protobuf 3, optional is default

Reply:
We still use proto2 in lake_service.proto now